apps/fxc-tiles/src/app/parser/openair.ts

```
@@ -227,6 +216,40 @@ export function parseAll(openair: string, country: string): Airspace[] {
+  endAngle: number,
+  coords: [lon: number, lat: number][],
+) {
+  if (direction === Direction.Clockwise && endAngle < startAngle) {
+    endAngle += 360;
+  }
+  if (direction === Direction.CounterClockwise && endAngle > startAngle) {
+    endAngle -= 360;
+  }
+  const angleStep = 10 * (direction === Direction.Clockwise ? 1 : -1);
+  // eslint-disable-next-line no-constant-condition
+  for (let i = 0; true; i++) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)